java/daikon/chicory/DaikonVariableInfo.java

```diff
@@ -832,7 +832,7 @@ protected DaikonVariableInfo addDeclVar(Field field, String offset, StringBuilde
         newField.repTypeName += " = " + value;
         newField.const_val = value;
         newField.dtraceShouldPrint = false;
-        if (DaikonVariableInfo.dkconfig_constant_infer && isPrimitive) {
+        if (dkconfig_constant_infer && isPrimitive) {
           newField.dtraceShouldPrintChildren = false;
         }
       }
```

Comment:
Github doesn't allow comments on unchanged lines?

Question is about line 895: "if (true)....", code after this is unreachable?